src/components/menu/Menu.tsx

```diff
@@ -199,7 +199,7 @@ class Menu extends React.Component<MenuProps> {
         const { menuItemEl }: { menuItemEl?: HTMLElement | null } =
             event.target instanceof Node ? this.getMenuItemElFromEventTarget(event.target) : {};
 
-        if (!menuItemEl) {
+        if (!menuItemEl || (event.target as Element).className.includes('submenu-target')) {
```

Comment:
Why is this change needed?

Reply:
SubmenuItem clicks are not distinguished from a MenuItem click and thus closes the menu when clicked on. This is a problem on mobile where users are not able to perform "hover" actions

![Dropdown-submenu-click-bug](https://user-images.githubusercontent.com/1041516/153464144-7a1e0889-39b1-4287-b46a-fcf69a48ee07.gif)